debian/rules

```diff
@@ -140,6 +140,10 @@ override_dh_python3:
 	@for pkg in $(py3_packages); do \
 	  dh_python3 -p $$pkg;          \
 	done
+	dh_python3 -p ceph-base --shebang=/usr/bin/python3
+	dh_python3 -p ceph-common --shebang=/usr/bin/python3
+	dh_python3 -p ceph-fuse --shebang=/usr/bin/python3
+	dh_python3 -p ceph-volume --shebang=/usr/bin/python3
```

Comment:
for the posterity, despite that ceph-volume comes with a `requirements_dev.txt`. it is not installed or participates in the packaging. the generated `DEBIAN/control` of ceph-volume looks like
```
Package: ceph-volume
Source: ceph
Version: 18.0.0-6344-g4fd6c24f-1focal
Architecture: all
Maintainer: Ceph Maintainers <ceph-maintainers@ceph.io>
Installed-Size: 724
Depends: ceph-osd (= 18.0.0-6344-g4fd6c24f-1focal), cryptsetup-bin, e2fsprogs, lvm2, parted, xfsprogs, python3:any
Section: python
Priority: optional
Homepage: http://ceph.com/
Description: tool to facilidate OSD deployment
 Ceph is a massively scalable, open-source, distributed
 storage system that runs on commodity hardware and delivers object,
 block and file system storage.
 .
 This package contains a tool to deploy OSD with different devices like
 lvm or physical disks, and trying to follow a predictable, and robust
 way of preparing, activating, and starting the deployed OSD.
```

Reply:
after auditing other packages' `DEBIAN/control` files, they all now have an additional `python3:any` dependency.